climada/entity/impact_funcs/test/test_tc.py

```diff
@@ -168,6 +168,46 @@ def test_get_countries_per_region(self):
         self.assertListEqual(out[2], [124, 840])
         self.assertListEqual(out[3], ["CAN", "USA"])
 
+    def test_get_region_per_countries(self):
+        """Test static get_regions_per_countries()"""
+        ifs = ImpfSetTropCyclone()
+        out = ifs.get_regions_per_countries(countries=["CHE"], code_type="ISO3A")
```

Comment:
Can you please use more clear variable names. `out` is not very clear here. Can you also make clear why the outputs are what they are?